src/app/task/done-tasks/done-tasks.component.html

```diff
@@ -1,12 +1,14 @@
-<h2> Done Tasks </h2>
+<h2 class="done-task-title"> Done Tasks </h2>
```

Comment:
Classes should be named by what they change. The name you used could be used as id not class name. For example if class changes text color to red, name would be text-red.